src/error.rs

```diff
@@ -213,6 +213,20 @@ impl ParseErrors {
     pub const fn none() -> ParseErrors {
         ParseErrors { errors: Vec::new() }
     }
+
+    pub fn from_recoverable<'a>(
+        errs: Vec<
+            lalrpop_util::ErrorRecovery<usize, parser::lexer::Token<'a>, parser::error::ParseError>,
```

Comment:
Nitpick: just for the sake of readability of the signature, it may look nicer to put the ugly type in a type alias.

Reply:
I imported `ErrorRecovery`, `Token` to make the signature more dense.